components/Slider/Slider.styled.js

```diff
@@ -2,23 +2,24 @@ import styled from "styled-components";
 
 export const IntensityContainer = styled.div`
   display: flex;
-  gap: 10px;
+  gap: var(--spacing-m);
 `;
 
 export const Intensity = styled.p`
   color: ${(props) =>
-    props.index === 1 ? "white" : "rgba(255, 255, 255, 0.5)"};
+    props.index === 1 ? `var(--color-main-alt)` : ` var(--color-neutral)`};
```

Comment:
I am not sure and please check this with a coach, but I think it would be more clear if you move the logic into the component. so check if index is 1 in the component and pass the prop into the styled component based on that

Reply:
thanks for that! implemented it